client/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-submit-button.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import debugFactory from 'debug';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	FormStatus,
+	useLineItems,
+	useEvents,
+	useFormStatus,
+	useSelect,
+} from '@automattic/composite-checkout';
+import { Button } from '@automattic/components';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+const debug = debugFactory( 'calypso:composite-checkout:credit-card' );
+
+export default function CreditCardSubmitButton( {
+	disabled,
+	onClick,
+	store,
+	stripe,
+	stripeConfiguration,
+	activeButtonText = undefined,
+} ) {
+	const { __ } = useI18n();
+	const dispatch = useDispatch();
+	const [ items, total ] = useLineItems();
+	const fields = useSelect( ( select ) => select( 'credit-card' ).getFields() );
+	const cardholderName = fields.cardholderName;
+	const cardholderEmail = fields.cardholderEmail;
+	const cardholderPhone = fields.cardholderPhone;
```

Comment:
Are `cardholderEmail` and `cardholderPhone` valid fields?
Looking at `createStripePaymentMethodStore` in `packages/composite-checkout/dist/esm/lib/payment-methods/stripe-credit-card-fields.js` then I can only find `cardholderName`.

PS: I might be looking in the wrong place though 😅  but it looks like we are dynamically trying to fetch values that I'm not sure is supposed to work like that. E.g.: should we use `fields.getCardholderName()` instead of `fields.cardholderName` ?